standalone/RL_plus_script.py

```diff
@@ -252,7 +253,7 @@ def main():
     # if `wandb_project_name` is set, the training logs and videos
     # will be uploaded to Weights and Biases
     # train()
-    # train(wandb_project_name="minerl")
+    train(wandb_project_name="minerl")
```

Comment:
leave this line commented out please

Reply:
ah sorry for the silly mistake; it's fixed now